src/components/asset-list/RecyclerAssetList/index.tsx

```diff
@@ -542,7 +492,7 @@ function RecyclerAssetList({
                 height: ViewTypes.COIN_SAVINGS.calculateHeight({
                   amountOfRows:
                     sections[balancesIndex].data[index - 1].assets?.length || 0,
-                  isLast: poolsIndex < 0,
+                  isLast: true,
```

Comment:
in the event of the Pools section not existing (`poolsIndex` is -1) then this would always be `true`